components/eam/src/chemistry/utils/read_spa_data.F90

```diff
@@ -0,0 +1,93 @@
+module read_spa_data
+
+!BSINGH - I didn't add any comments here due to time constrain but I will add comments later
+
+use tracer_data
```

Comment:
Use `only` with the `use` statement

Reply:
I think you just need only: trfld,trfile at the module level since individual functions have their own `use tracer_data, only` commands.